src/components/Footer/Footer.tsx

```diff
@@ -14,21 +14,21 @@ const Footer: React.FC = () => {
   );
 };
 
-const StyledFooter = styled.footer`
-  align-items: center;
-  display: flex;
-  justify-content: center;
-  flex-direction: column;
-  text-align: center;
-  width: 100%;
-`;
+const StyledFooter = styled("footer")(({ theme }) => ({
+  "align-items": "center",
```

Comment:
this will work for now but i meant for these to become the css camelcase keys so that we'd get more protection of their values from typescript.

Reply:
I think I fixed them all to the camelcase method now